src/database/schema_metadata.py

```
@@ -1142,7 +1142,15 @@ def _schema_backed_metadata(
             {"name": "父父繁殖登録番号", "type": "TEXT", "description": "父父（父の父）繁殖登録番号", "example": "1111111111", "nullable": True},
             {"name": "父母繁殖登録番号", "type": "TEXT", "description": "父母（父の母）繁殖登録番号", "example": "2222222222", "nullable": True},
             {"name": "母父繁殖登録番号", "type": "TEXT", "description": "母父（母の父）繁殖登録番号", "example": "3333333333", "nullable": True},
-            {"name": "母母繁殖登録番号", "type": "TEXT", "description": "母母（母の母）繁殖登録番号", "example": "4444444444", "nullable": True}
+            {"name": "母母繁殖登録番号", "type": "TEXT", "description": "母母（母の母）繁殖登録番号", "example": "4444444444", "nullable": True},
```

Ruff (0.16.1)

[warning] String contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF001)